index.js

```diff
@@ -40,8 +40,11 @@ class ExecutorRouter extends Executor {
             const options = Object.assign({ ecosystem }, plugin.options); // Add ecosystem to executor options
             const executorPlugin = new ExecutorPlugin(options);
 
-            // Set the default executor
-            if (!this.defaultExecutor && !config.defaultPlugin) {
+            if (defaultPlugin && defaultPlugin === plugin.name) {
```

Comment:
Oh I meant on this if statement. Just for shorter failing out of the `if` statement